packages/relay/src/lib/eth.ts

```diff
@@ -580,21 +582,20 @@ export class EthImpl implements Eth {
       }
     }
 
-    // check cache first
-    // create a key for the cache
-    const cacheKey = `${constants.CACHE_KEY.ETH_GET_BALANCE}-${account}-${blockNumberOrTag}`;
-    const cachedBalance = this.cache.get(cacheKey);
-    if (cachedBalance) {
-      this.logger.trace(`${requestIdPrefix} returning cached value ${cacheKey}:${JSON.stringify(cachedBalance)}`);
-      return cachedBalance;
-    }
-
     let blockNumber = null;
     let balanceFound = false;
     let weibars: BigInt = BigInt(0);
     const mirrorAccount = await this.mirrorNodeClient.getAccount(account, requestId);
 
     try {
+      // check balance cache first for high load scenarios
```

Comment:
Move cache check to the top after `getBalance(account=` log

Reply:
Done.